buildpipeline/DotNet-CoreRT-Windows.json

```diff
@@ -477,7 +477,7 @@
       "fetchDepth": "0",
       "gitLfsSupport": "false",
       "skipSyncSource": "false",
-      "cleanOptions": "0"
```

Comment:
Do the same on Unix and Mac?

Reply:
+1 for Jan's comment.  We didn't notice the low disk space issue in non-Windows machines since DDIT has an auto-cleaning mechanism

For context, setting to `3` would delete all build directories. Refer to https://docs.microsoft.com/en-us/vsts/build-release/concepts/definitions/build/repository

> All build directories: Deletes and recreates $(Agent.BuildDirectory).